### PR TITLE
Use fakerConfig dsl instead of create fun

### PR DESCRIPTION
**Pull request by johanvergeer**
I've used the Kotlin dsl instead of the `create()` method.

### CODE REVIEW

File: core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt [hunk 1]
```diff
@@ -60,9 +54,12 @@ class FakerConfig private constructor(
 
 @Deprecated(
     message = "This function is deprecated and will be removed in future releases",
-    ReplaceWith("FakerConfig.create { }"),
+    ReplaceWith("fakerConfig { ]"),
```
Comment (serpro69) [inline core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt:57]: A typo with `]`, should be `}` instead.

**Review by serpro69 — CHANGES_REQUESTED** [review]
Can you please fix the typo in the "ReplaceWith" part of the `Deprecated` annotation. (See inline comment)
Otherwise looks fine.

**Comment by serpro69** [comment]
It looks like compilation is also failing in the `cli-bot` module:
```
e: /home/travis/build/serpro69/kotlin-faker/cli-bot/src/main/kotlin/io/github/serpro69/kfaker/app/subcommands/List.kt: (37, 39): Unresolved reference: create
e: /home/travis/build/serpro69/kotlin-faker/cli-bot/src/main/kotlin/io/github/serpro69/kfaker/app/subcommands/List.kt: (38, 13): Unresolved reference: locale
e: /home/travis/build/serpro69/kotlin-faker/cli-bot/src/main/kotlin/io/github/serpro69/kfaker/app/subcommands/Lookup.kt: (32, 39): Unresolved reference: create
e: /home/travis/build/serpro69/kotlin-faker/cli-bot/src/main/kotlin/io/github/serpro69/kfaker/app/subcommands/Lookup.kt: (33, 13): Unresolved reference: locale
```

File: core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt [hunk 2]
```diff
@@ -46,7 +40,7 @@ class FakerConfig private constructor(
      * Default: `100`.
      */
     @FakerDsl
-    class Builder internal constructor() {
```
Comment (serpro69) [inline core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt:49]: Any reason to make the constructor public?

**Review by serpro69 — CHANGES_REQUESTED** [review]
Would also be good to make changes to the readme: https://github.com/serpro69/kotlin-faker#configuring-faker

File: core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt (hunk 2 above)
Comment (johanvergeer) [reply core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt:49]: Yes. It's not possible to make the class `internal` or `private` when it's used as a parameter.

image

Comment (johanvergeer) [reply core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt]: Good point. I made them internal again. I guess I overlooked them.

File: core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt (hunk 1 above)
Comment (johanvergeer) [reply core/src/main/kotlin/io/github/serpro69/kfaker/FakerConfig.kt]: Fixed

**Review by serpro69 — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**